### ISSUE TITLE
Enhancement req: Support Sponsorblock minimum duration

### ISSUE THREAD

**Issue by rodalpho** [yuliskov/SmartTubeNext#1662]
The sponsorblock browser addon has a setting to ignore skips of less than a configurable number of seconds. I find this useful because many segments are extremely short, like 2-3 seconds, and the buffering YouTube time makes skipping them actually slower than just watching the very brief mention of a sponsor. Can this be added to the app?

![image](https://user-images.githubusercontent.com/747832/177020576-e042df89-08f4-438f-86f7-e894c9d57207.png)

**Comment by Kinkyboo** [comment]
Agree, it is super annoying when some people marks ever single mention of a merchstore / self-promo

**Comment by albino1** [comment]
Frequently lately people have been flagging even chapter markers or title cards.  For example, if it's a travelogue and the channel includes 3 seconds of text on screen saying which destination they've arrived at, somebody will mark that as a Title Card/Intro in SponsorBlock as if that information is something somebody might want to skip.  The constant popups to skip these is frequently more annoying than any actual sponsored segment.
